apps/ios/react-native.config.js

```diff
@@ -15,19 +15,10 @@
  */
 'use strict';
 
-const macSwitch = '--use-react-native-macos';
```

Comment:
Are you sure this is safe to delete?

Reply:
We aren't using any of the forks for react-native in the iOS app, so yes?

Reply:
True, it's really the macOS app we may need it

Reply:
Shouldn't we be very much be using the fork for the iOS app so fluentui-react-native is using the same version as other Microsoft experiences using react-native? Or am I missing something?

Reply:
When we hooked up the iOS app to the lerna system and moved it one level up out of `apps/apple/`, we switched from using the macOS fork to the facebook master version. Given that iOS is not an out of tree platform at the moment, this feels the more correct apporach to me. Technically, `react-native-mac`, `react-native-win32`, and `react-native-windows` all can build iOS apps, and are all "Microsoft Experiences". I didn't make the decision to use `react-native` over `react-native-macos`, but it feels like the more correct approach to me.

Reply:
An advantage of using react-native-macos is that we can more quickly iterate if we need a one-off bug fix. I picture the ideal world as us depending directly on react-native (master) and we have the ability to make changes in that repo quickly